lms/djangoapps/course_goals/management/commands/goal_reminder_email.py

```diff
@@ -51,7 +51,7 @@ def send_ace_message(goal):
     site = Site.objects.get_current()
     message_context = get_base_template_context(site)
 
-    course_home_url = get_learning_mfe_home_url(course_key=goal.course_key)
+    course_home_url = get_learning_mfe_home_url(course_key=goal.course_key, view_name='home')
```

Comment:
What's the reason for this change?

Reply:
without this it was sending people to https://learning.edx.org/course/course-v1:GTx+MGT6203x+1T2021/ without the home at the end

Reply:
👍